collector/generatorreceiver/topos/hipster_shop.yaml

```diff
@@ -14,16 +33,14 @@ topology:
       resourceAttrSets:
         - weight: 1
           resourceAttrs:
-            cloud.provider: aws
-            cloud.region: us-east-1
             k8s.cluster.name: k8s-cluster-1
+            <<: *aws.east1
```

Comment:
I wonder if you have thoughts/opinions on defining cloud providers separately, for example:
```yaml 
environments:
  - name: "azure-central-us"
    attributes:
      cloud.provider: azure
      cloud.region: Central-US
```

And then we would set an `environment` field for each service:

```yaml 
- serviceName: frontend
  environment: "azure-central-us"
  tagSets:
    - weight: 1
      tags:
        version: v127
     ...
```

The generator  would process and join the environment attributes with service attributes when loading the config.

Another thing that this could be helpful is to generate system metrics about a service. For example, if we follow this "environment" approach, we could use that info, to create fake hosts for the services(e.g., in the example above, we could create a host `azure-central-us-prod-1`) and emit some cpu and memory metrics.